c_plus_plus/src/model/audio/i_audio_sink.h

```diff
@@ -0,0 +1,19 @@
+#ifndef I_AUDIO_SINK_H
+#define I_AUDIO_SINK_H
+
+
+namespace Model
+{
+
+class IAudioSink
+{
+public:
+    virtual ~IAudioSink() {}
+    virtual bool open() = 0;
+    virtual bool close() = 0;
+    virtual int write(uint8_t* buffer, int nbytes) = 0;
```

Comment:
We should include the lib string or use forward declaration because uint8_t is undefined in this header file.